src/sources/webgl.ts

```diff
@@ -132,7 +133,8 @@ export function getWebGlExtensions({ cache }: Options): WebGlExtensionsPayload |
   // Extension parameters
   if (extensions) {
     for (const name of extensions) {
-      if (name === rendererInfoExtensionName && shouldAvoidDebugRendererInfo()) {
+      // The "polygon mode" extension causes a console warning in Chromium and WebKit
```

Comment:
Firefox emits no warning because it doesn't support `WEBGL_polygon_mode` yet (even the nightly 124 version). When it does support, it's very likely to have a similar warning too.